### PR TITLE
Create user

### PR DESCRIPTION
**Pull request by TeslaRobotics**
Add user controller and rspec tests

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  def index
+    render json: User.all
+  end
+
+  def show
+    render json: User.find(params[:id])
+  end
+
+  def create
+    user_params = params.slice(:email, :password, :display_name, :full_name, :biography, :phone).to_unsafe_hash
```
Comment (condef5) [inline app/controllers/users_controller.rb]: What does this method do?

File: spec/controllers/users_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+      expect(expected_user["id"]).to eq(user.id)
+    end
+
+    it 'returns http status not found' do
+      get :show, params: { id: 'xxx' }
+      expect(response).to have_http_status(:not_found)
+    end
+  end
+
+#  TODO: nos quedamos aqui
```
Comment (condef5) [inline spec/controllers/users_controller_spec.rb]: Remove this unnecesary comment

File: app/controllers/users_controller.rb (hunk 1 above)
Comment (TeslaRobotics) [reply app/controllers/users_controller.rb]: params is an instance of the class `ActionController::Parameters` an that class have the method `to_unsafe_hash` to transform it to a hash, so we can use it to create a User.

File: spec/controllers/users_controller_spec.rb (hunk 2 above)
Comment (TeslaRobotics) [reply spec/controllers/users_controller_spec.rb]: Ok

**Comment by condef5** [comment]
Thanks, good job

**Pull request merged**